src/aspire/utils/filters.py

```diff
@@ -305,7 +322,9 @@ def _evaluate(self, omega):
         return h.squeeze()
 
     def scale(self, c=1):
-        self.pixel_size *= c
+        new_filter = ScaledFilter(copy.copy(self), c)
```

Comment:
We want to return a `CTFFilter` with the correct pixel size here, not a `ScaledFilter` (otherwise why even override this?)

Reply:
Changed to return new CTFFilter instead of ScaledFilter in new commit 6321619.